src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs

```diff
@@ -0,0 +1,108 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Garnet;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Garnet resources to the application model.
+/// </summary>
+public static class GarnetBuilderExtensions
+{
+    /// <summary>
+    /// Adds a Garnet container to the application model.
+    /// </summary>
+    /// <remarks>
+    /// The default image is "garnet" and the tag is "1.0.2".
```

Comment:
Remove this remark, I think it'll be a pain to maintain this.

Reply:
This will indeed be difficult to maintain. Deleted